tower-fallback/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tower-fallback"
-version = "0.2.40"
+version = "0.2.41"
```

Comment:
I think the tower crates should be `beta` versions, because we don't support them and their API isn't stable.

Using these `cargo release` commands should do this automatically:
https://github.com/ZcashFoundation/zebra/pull/6909/files#r1226047767

Reply:
I did this in PR #6908.